opm/material/fluidsystems/blackoilpvt/BrineCo2Pvt.hpp

```diff
@@ -738,4 +793,44 @@ class BrineCo2Pvt
 
 } // namespace Opm
 
+namespace Opm::gpuistl {
+    template<class Scalar, class Params, class GPUContainer>
+    BrineCo2Pvt<Scalar, Params, GPUContainer>
+    move_to_gpu(BrineCo2Pvt<Scalar> cpuBrineCo2) {
```

Comment:
Should be const ref? Prefer brace on new line for functions and classes, also for make_view().